src/main/java/com/photolocos/enterprise/service/IPhotoService.java

```diff
@@ -7,16 +7,41 @@
 import java.util.Set;
 
 public interface IPhotoService {
-
-    PhotoDTO fetchByLocation(String location);
-
+    /**
+     * Fetches a photo using location name
+     *
+     * @param locationId
+     */
+    PhotoDTO fetchByLocation(String locationId);
+
+    /**
+     * Fetches photos using descriptive tag
+     *
+     * @param tag
+     */
     Set<PhotoDTO> fetchByTag(String tag);
 
+    /**
+     * Saves a photo to the database
+     *
+     * @param photo
+     * @param image
```

Comment:
If adding JavaDoc, complete the @param and @return annotations.  the comment on most of these methods (saves a photo to the database) can be read from the method name itself (savePhoto(PhotoDTO)), and thus, isn't adding much value, on its own.